gradle/libs.versions.toml

```diff
@@ -0,0 +1,152 @@
+android-junit5 = { module = "de.mannodermaus.gradle.plugins:android-junit5", version.ref = "androidJunit5" }
+androidx-compose-foundation = { module = "androidx.wear.compose:compose-foundation", version.ref = "composeFoundation" }
+androidx-compose-material = { module = "androidx.wear.compose:compose-material", version.ref = "composeFoundation" }
+androidx-compose-navigation = { module = "androidx.wear.compose:compose-navigation", version.ref = "composeFoundation" }
+androidx-health-services-client = { module = "androidx.health:health-services-client", version.ref = "healthServicesClient" }
+androidx-room-compiler = { module = "androidx.room:room-compiler", version.ref = "roomRuntime" }
+androidx-room-ktx = { module = "androidx.room:room-ktx", version.ref = "roomKtx" }
+androidx-room-runtime = { module = "androidx.room:room-runtime", version.ref = "roomRuntime" }
+androidx-lifecycle-viewmodel-ktx = { module = "androidx.lifecycle:lifecycle-viewmodel-ktx", version.ref = "lifecycleViewmodelKtx" }
+androidx-tiles-material = { module = "androidx.wear.tiles:tiles-material", version.ref = "tilesMaterial" }
+androidx-tiles = { module = "androidx.wear.tiles:tiles", version.ref = "tiles" }
+androidx-watchface-complications-data-source-ktx = { module = "androidx.wear.watchface:watchface-complications-data-source-ktx", version.ref = "watchfaceComplicationsDataSourceKtx" }
+androidx-work-runtime-ktx = { module = "androidx.work:work-runtime-ktx", version.ref = "workRuntimeKtx" }
+app-car = { module = "androidx.car.app:app", version.ref = "car-app" }
+app-automotive = { module = "androidx.car.app:app-automotive", version.ref = "car-app" }
+app-projected = { module = "androidx.car.app:app-projected", version.ref = "car-app" }
+appcompat = { module = "androidx.appcompat:appcompat", version.ref = "appcompat" }
+biometric = { module = "androidx.biometric:biometric", version.ref = "biometric" }
+blurView = { module = "com.github.Dimezis:BlurView", version.ref = "blurView" }
+changeLog = { module = "com.github.AppDevNext:ChangeLog", version.ref = "changeLog" }
+community-material-typeface = { module = "com.mikepenz:community-material-typeface", version.ref = "community-material-typeface" }
+compose-bom = { module = "androidx.compose:compose-bom", version.ref = "compose-bom" }
+compose-animation = { module = "androidx.compose.animation:animation" }
+compose-foundation = { module = "androidx.compose.foundation:foundation" }
+compose-material = { module = "androidx.compose.material:material" }
+compose-material-iconsCore = { module = "androidx.compose.material:material-icons-core" }
+compose-material-iconsExtended = { module = "androidx.compose.material:material-icons-extended" }
+compose-runtime = { module = "androidx.compose.runtime:runtime" }
+compose-ui = { module = "androidx.compose.ui:ui" }
+compose-uiTooling = { module = "androidx.compose.ui:ui-tooling" }
+compiler = { module = "androidx.compose.compiler:compiler", version.ref = "compose-compiler" }
```

Comment:
Can we add Compose to the name? Compiler is very general

Reply:
Fixed